src/Cli/Microsoft.TemplateEngine.Cli/Commands/create/InstantiateCommand.cs

```diff
@@ -434,8 +453,7 @@ async Task<string> GetTemplatePackage(CliTemplateInfo template)
                     template,
                     validateDefaultLanguage);
 
-                CliConfiguration parser = ParserFactory.CreateParser(command);
-                ParseResult parseResult = parser.Parse(args.RemainingArguments ?? Array.Empty<string>());
+                ParseResult parseResult = Reparse(command, args.RemainingArguments ?? Array.Empty<string>());
```

Comment:
You no longer need to create a parser out of the factory and sent the command to it. The `Parse` method on the command (which may or may not be an extension method) does this already.